protocol/README.md

```diff
@@ -4,12 +4,11 @@ This document describes the canonical system architecture used to encrypt and de
 
 ## Architecture
 
-![arch](../diagrams/arch.png)
-
-The canonical architecture contains three major components.
+The canonical architecture contains four major components.
 
 * *TDF Client* - Initiates and drives the TDF encryption and decryption workflows. Only component with access to the content (ciphertext or plaintext).
-* *Entity Attribute Service* (EAS) - Responsible for authenticating TDF Clients and vending them signed [Entity Objects](../schema/EntityObject.md), which act as bearer tokens. The source of truth for attribute provenance and association. A single architecture can have many EAS instance running, for instance, for a TDF Client to join attributes across mulitple orginizations.
+* *Virtru Identity Provider (IdP) (Keycloak)* - From Wikipedia:  "Keycloak is an open source software product to allow single sign-on with Identity and Access Management aimed at modern applications and services. As of March 2018 this JBoss community project is under the stewardship of Red Hat who use it as the upstream project for their RH-SSO product."  Keycloak is open source Apache License 2.0 code.
```

Comment:
Can we talk about how it works with OIDC and IdPs in a generic term for the spec?

Reply:
Yeah - see below for (a) suggestion